src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1453,6 +1453,23 @@ void ICanSaveModel.Save(ModelSaveContext ctx)
            bool trimWhitespace = Defaults.TrimWhitespace,
            IMultiStreamSource dataSample = null)
         {
+            Options options = new Options
+            {
```

Comment:
Is the plan to add readMultiLines as an option in a later PR?

Reply:
`readMultilines` is already added to the TextLoader.Options class in my other PR, and it will always default as false, if its value isn't provided.

The method here, that you've pointed to, is used by [this public method](https://github.com/dotnet/machinelearning/blob/212081c62855a835e0612887f705a8f87a899624/src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs#L112-L120) in the catalog. I can't add a new readMultilines parameter there precisely because it would be a breaking API change. So it doesn't make much sense to me to add a readMultilines option here, and so TextLoader.Options will simply use the default for it (which is false).

In the future, if a user (or we internally) want to use the new options (such as readMultiline), we would simply use a TextLoader.Options object and the overloads for creating TextLoaders that uses the option object. I think it's better and easier to maintain and update only the options object than an overload with several parameters that can't be changed in the public API.